index.html

```diff
@@ -201,5 +201,19 @@ <h2 class="aheading">About Myself</h2>
   </div>
         </div>
       </section>
+      <div >
+      <footer class="contact-form"> 
+      <form action="https://formspree.io/f/{form_id}" method="post">
+        <ul class="form_items">
+<li class="form_items_1"> <h1 class="contact-title">Contact me</h1></li>
+<li class="form_items_1"><p class="contact-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p></li>
+<li class="form_items_1"><input class="formtext" type="text" name="name" placeholder="Burak" maxlength="30" required></li>   
+<li class="form_items_1"><input class="formemail" type="email" name="email" placeholder="r.brk.otman@gmail.com" required></li>
+<li class="form_items_1"><textarea class="formtextarea" placeholder="Write your message here" maxlength="500" required></textarea></li>    
```

Comment:
- [ ] Good job using the `required` attribute to validate the input fields. Kindly put the `name` attribute on the `textarea` element so its content can be collected by the Formspree service. 🤓️